lib/algoliax/indexer.ex

```diff
@@ -132,19 +132,95 @@ defmodule Algoliax.Indexer do
-      "query" => "john"
-      }}
+          "hitsPerPage" => 20,
+          "nbHits" => 16,
+          "nbPages" => 1,
+          "page" => 0,
+          "params" => "query=john",
+          "processingTimeMS" => 1,
+          "query" => "john"
+        }
+      }
   """
 
   @callback search(query :: binary(), params :: map()) ::
               {:ok, Algoliax.Response.t()} | {:not_indexable, model :: map()}
 
+  @doc """
+  Search for multiple queries
+
+  ## Example
+
+      iex> People.search_multiple([%{indexName: "index_1", params: %{query: "jon"}}, %{indexName: "index_2", params: %{query: "bar"}}])
+      {:ok,
+        %{
+          response: %{
+            "results" => [
+              %{
+                "hits" => [
+                  %{
+                    "_highlightResult" => %{
+                      "full_name" => %{
+                        "fullyHighlighted" => false,
+                        "matchLevel" => "full",
+                        "matchedWords" => ["jon"],
+                        "value" => "Pierre <em>Jon</em>es"
+                      }
+                    },
+                    "age" => 69,
+                    "first_name" => "Pierre",
+                    "full_name" => "Pierre Jones",
+                    "indexed_at" => 1570908223,
+                    "last_name" => "Jones",
+                    "objectID" => "b563deb6-2a06-4428-8e5a-ca1ecc08f4e2"
+                  },
+                  ...
+                ],
+                "index" => "index_1",
+                "hitsPerPage" => 20,
+                "nbHits" => 16,
+                "nbPages" => 1,
+                "page" => 0,
+                "params" => "query=jon",
+                "processingTimeMS" => 1,
+                "query" => "jon"
+              },
+              %{
+                "hits" => [
+                  %{
+                    "_highlightResult" => %{
+                      "full_name" => %{
+                        "fullyHighlighted" => false,
+                        "matchLevel" => "full",
+                        "matchedWords" => ["bar"],
+                        "value" => "Glennie <em>Bar</em>nes"
+                      }
+                    },
+                    "age" => 27,
+                    "first_name" => "Glennie",
+                    "full_name" => "Glennie Barnes",
+                    "indexed_at" => 1570908223,
+                    "last_name" => "Barnes",
+                    "objectID" => "58e8ff8d-2794-41e1-a4ef-6f8db8d432b6"
+                  },
+                ],
+                "index" => "index_2",
+                "hitsPerPage" => 20,
+                "nbHits" => 16,
+                "nbPages" => 1,
+                "page" => 0,
+                "params" => "query=bar",
+                "processingTimeMS" => 1,
+                "query" => "bar"
+              }
+            ]
+          }
+        }
+      }
+  """
+  @callback search_multiple(queries :: list(map()), strategy :: String.t()) ::
```

Comment:
Should this be attached to an Algolia resource at all? Why not just a generic submodule under `Algoliax`? We’ll be using this to query e.g. playlists alongside recent songs, which will have separate Algoliax definitions

Reply:
Good call. I'll move it out from here.